tests/srp/rc

```diff
@@ -51,7 +51,13 @@ group_requires() {
 	_have_module ib_uverbs
 	_have_module null_blk
 	_have_module rdma_cm
-	_have_module rdma_rxe
+	if [ -n "$use_rxe" ]; then
+		_have_module rdma_rxe
+	else
+		_have_module siw
+		_have_kver 5 5
+		_have_iproute2 190404
```

Comment:
I think this line is ok, but l leave a minor comment: ip tool changed its versioning format since 2022 August: [ref](https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/?id=fbef655568ee931a82ad463a6f46f01ce3fb27aa). Until that version, _have_iproute2() works as expected.  After that _have_iproute2() does not check version exactly as expected, but it just does nothing. No side effect . So, I think this line is OK. _have_iproute2 can be improve to support both version strings before and after 2022 August. It will be a low priority left work.